manifest.json

```diff
@@ -36,7 +36,7 @@
         "favicon_url": "https://duckduckgo.com/favicon.ico",
         "is_default": true,
         "keyword": "duckduckgo.com",
-        "name": "DuckDuckGo Search",
```

Comment:
This was redundant as Chrome will automatically append the word "Search" to the search provider's name, resulting in labels like "DuckDuckGo Search Search" in the omnibox.